src/server/rest/v1/service/security/CarSecurity.ts

```diff
@@ -264,6 +264,7 @@ export default class CarSecurity {
         type: car.type,
         converter: car.converter,
         carCatalog: car.carCatalog,
+        carUsers: car.carUsers
```

Comment:
Use the User filter method instead.

Reply:
we already use the filterMinimalUserResponse in the line after this (same as carCatalog)

Reply:
Okay :)